src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -225,14 +243,12 @@ const styles = StyleSheet.create({
   }),
   navContainer: (isHeaderHide, top) => ({
     flexDirection: 'row',
-    height:
-      (isHeaderHide
-        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
-        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + top,
+    height: isHeaderHide
+      ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
+      : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT,
     justifyContent: 'space-between',
     alignItems: 'flex-start',
     backgroundColor: 'white',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `navContainer` style seems to remove the padding at the top. This could potentially cause layout issues, especially on iOS devices with a notch. Please verify that the layout still looks correct on these devices after this change.
```diff
- paddingTop: dimen.normalizeDimen(12) + top,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->